test/box-luatest/gh_8036_create_space_with_explicit_and_implicit_id_test.lua

```diff
@@ -0,0 +1,63 @@
+local server = require('luatest.server')
+local t = require('luatest')
+
+local g = t.group('Create space with explicit and implicit id', {
+    {sql = false},
+    {sql = true}
+})
+
+g.before_each(function(cg)
+    cg.server = server:new{
+        alias   = 'default',
+        box_cfg = {}
+    }
+    cg.server:start()
+end)
+
+g.after_each(function(cg)
+    cg.server:drop()
```

Comment:
Recreating the test server after each test case extends the test time. We should avoid it if possible. In this case, it'd be better to drop the test spaces instead.